src/raftstore/store/snap.rs

```diff
@@ -2493,64 +1938,8 @@ mod test {
     }
 
     #[test]
-    fn test_snap_v1_v2_compatible() {
-        // Ensure that it's compatible between v1 impl and v2 impl, which means a receiver
-        // running v2 code could receive and apply the snapshot sent from a sender running v1 code.
-        let src_temp_dir = TempDir::new("test-snap-v1-v2-compatible-src").unwrap();
-        let src_path = src_temp_dir.path().to_str().unwrap().to_owned();
-        let src_mgr = SnapManager::new(src_path, None, false);
-        src_mgr.init().unwrap();
-
-        let src_db_dir = TempDir::new("test-snap-v1-v2-compatible-src-db").unwrap();
-        let db = v2::test::get_test_db(&src_db_dir).unwrap();
-        let snapshot = DbSnapshot::new(db.clone());
-        let key = SnapKey::new(1, 1, 1);
-        let mut s1 = src_mgr.get_snapshot_for_building(&key, &snapshot).unwrap();
-        let region = v2::test::get_test_region(1, 1, 1);
-        let mut snap_data = RaftSnapshotData::new();
-        snap_data.set_region(region.clone());
-        let mut stat = SnapshotStatistics::new();
-        s1.build(&snapshot,
-                   &region,
-                   &mut snap_data,
-                   &mut stat,
-                   Box::new(src_mgr.clone()))
-            .unwrap();
-        let mut v = vec![];
-        snap_data.write_to_vec(&mut v).unwrap();
-
-        let mut s2 = src_mgr.get_snapshot_for_sending(&key).unwrap();
-        let expected_size = s2.total_size().unwrap();
-
-        let dst_temp_dir = TempDir::new("test-snap-v1-v2-compatible-dst").unwrap();
-        let dst_path = dst_temp_dir.path().to_str().unwrap().to_owned();
-        let dst_mgr = SnapManager::new(dst_path, None, true);
-        dst_mgr.init().unwrap();
-
-        let mut s3 = dst_mgr.get_snapshot_for_receiving(&key, &v[..]).unwrap();
-        let n = io::copy(&mut s2, &mut s3).unwrap();
-        assert_eq!(n, expected_size);
-        s3.save().unwrap();
-
-        let dst_db_dir = TempDir::new("test-snap-v1-v2-compatible-dst-db").unwrap();
-        let dst_db = Arc::new(rocksdb::new_engine(dst_db_dir.path().to_str().unwrap(), ALL_CFS)
-            .unwrap());
-        let options = ApplyOptions {
-            db: dst_db.clone(),
-            region: region.clone(),
-            abort: Arc::new(AtomicUsize::new(JOB_STATUS_RUNNING)),
-            write_batch_size: 10 * 1024 * 1024,
-        };
-        let mut s4 = dst_mgr.get_snapshot_for_applying(&key).unwrap();
-        s4.apply(options).unwrap();
-
-        v2::test::assert_eq_db(db, dst_db.as_ref());
-    }
-
-    #[test]
     fn test_snap_deletion_on_registry() {
-        test_snap_deletion_on_registry_impl(false);
-        test_snap_deletion_on_registry_impl(true);
+        test_snap_deletion_on_registry_impl();
```

Comment:
Rename `test_snap_deletion_on_registry_impl()` to `test_snap_deletion_on_registry()` directly.